library/core/src/main/java/com/google/android/exoplayer2/text/subrip/SubripDecoder.java

```diff
@@ -95,8 +128,36 @@ protected SubripSubtitle decode(byte[] bytes, int length, boolean reset) {
         textBuilder.append(currentLine.trim());
       }
 
-      Spanned text = Html.fromHtml(textBuilder.toString());
-      cues.add(new Cue(text));
+      // Extract tags
+      SubtitleTagResult tagResult = extractTags(textBuilder);
+      Spanned text = Html.fromHtml(tagResult.cue);
+
+      Cue cue = null;
+
+      // Check if tags are present
+      if (tagResult.tags.length > 0) {
+
+        boolean alignTagFound = false;
+
+        // At end of this loop the clue must be created with the applied tags
+        for (String tag : tagResult.tags) {
+
+          // Check if the tag is an alignment tag
+          if (tag.matches(SUBRIP_ALIGNMENT_TAG)) {
+
+            // Based on the specs, in case of the alignment tags only the first appearance counts
+            if (alignTagFound) continue;
+            alignTagFound = true;
+
+            AlignmentResult alignmentResult = getAlignmentValues(tag);
```

Comment:
If you change `getAlignmentValues(tag)` to be `buildCue(Spanned text, String alignmentTag)` then you can get rid of `AlignmentResult`.